libs/novui/src/tokens/semanticRadius.tokens.ts

```diff
@@ -1,9 +1,9 @@
 import { defineSemanticTokens } from '@pandacss/dev';
 
 /**
- * Represents the size of an element.
+ * Represents the size of an radius.
  *
- * Used for properties like width and height.
+ * Used for properties like radius.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do not leave false praise (which can actually be damaging). Do look for something to sincerely praise.  -->
👏 **praise:** Good catch, thank you!‏